Unicode/utype.c

```diff
@@ -41001,7 +41001,7 @@ const uint32 ____utype[] = { 0,
   0x180c2010, 0x00080000, 0x00004000, 0x00004000, 0x00004000, 0x00004000, 0x00000000, 0x00000000,
   0x10000000, 0x00000000, 0x10200100, 0x00000000, 0x00000000, 0x00020000, 0x00000000, 0x10000000,
   0x00004000, 0x00004000, 0x10000400, 0x10000400, 0x10010000, 0x10200101, 0x00000000, 0x00000000,
-  0x10000000, 0x10000400, 0x10200100, 0x00000000, 0x10000000, 0x10000000, 0x10000000, 0x00080000,
+  0x10000000, 0x10000400, 0x10200100, 0x00000000, 0x30000000, 0x30000000, 0x30000000, 0x00080000,
```

Comment:
So this table is a bit-per-field mask, and we're interested in more codepoints than before.